lib/chef/provisioning/convergence_strategy/install_msi.rb

```diff
@@ -36,7 +37,8 @@ def setup_convergence(action_handler, machine)
         super
 
         install_command = Mixlib::Install.new(chef_version, true, opts).install_command
-        machine.execute(action_handler, install_command)
+        machine.write_file(action_handler,convergence_options[:install_script_path],install_command)
+        machine.execute(action_handler, "& '#{convergence_options[:install_script_path]}'")
```

Comment:
something like `powershell.exe -ExecutionPolicy Unrestricted -NoProfile '#{convergence_options[:install_script_path]}'` seems to work better, and avoids ExecutionPolicy problems.